react/components/MiniCartContent.js

```diff
@@ -124,8 +124,8 @@ class MiniCartContent extends Component {
   }
 
   renderWithoutItems = label => (
-    <div className="vtex-minicart__item pa9 flex items-center justify-center relative bg-white">
-      <span className="f5">{label}</span>
+    <div className="vtex-minicart__item pa9 flex items-center justify-center relative bg-base">
+      <span className="t-action">{label}</span>
```

Comment:
I think that it should be t-body, since the element doesn't represent an action

Reply:
yeah, you're right. `t-body` has more semantic meaning :)